posts/2022-06-06-introduction-cypress-en.adoc

```diff
@@ -0,0 +1,216 @@
+= Introducing Cypress
+VincentBrule
+v1.0, 2022-06-06
+:title: Introducing Cypress
+:imagesdir: ../media/2022-06-06-introduction-cypress
+:lang: en
+:tags: [test, cypress, web, js, ts]
+
+This article is the beginning of a series of three articles dedicated to https://www.cypress.io[Cypress]:
+
+. Introducing Cypress
+. Going Further with Cypress
+. Comparing Cypress and its Competitors
+
+The goal of this first article is to make you want to discover and use *Cypress*. 
```

Comment:
The goal of this first article is to encourage you to discover and use *Cypress*.